crates/milli/src/update/new/indexer/document_operation.rs

```diff
@@ -297,13 +320,25 @@ fn extract_addition_payload_changes<'r, 'pl: 'r>(
                 },
                 Entry::Vacant(entry) => match method {
                     ReplaceDocuments => {
+                        if payload_operations.is_new
+                            && matches!(on_missing_document, MissingDocumentPolicy::Skip)
+                        {
+                            continue;
+                        }
```

Comment:
Can we also move this part before the `match method`, so that we have it only once in the `Entry::Vacant` branch, please?